Library/Basic Set/Basic Set Skills.skl

```diff
@@ -868,6 +868,35 @@
 					"name": "Engineer",
 					"specialization": "Body Armor",
 					"modifier": -4
+				},
+				{
+					"type": "skill",
+					"name": "Smith",
+					"specialization": "Bronze",
+					"modifier": -3,
+					"when_tl": {
+						"compare": "is",
+						"qualifier": 1
+					}
+				},
+				{
+					"type": "skill",
+					"name": "Smith",
+					"specialization": "Iron",
+					"modifier": -3,
+					"when_tl": {
+						"compare": "at_most",
```

Comment:
Technically this should be TL2-4 rather than TL <= 4 but I don't think GCS supports that currently, so this is probably the best we can do.